lib/repo/subversion_repository.rb

```diff
@@ -565,44 +561,56 @@ def expand_path(file_name, dir_string = "/")
     ##  this class).
     ####################################################################
 
-    # Semi-private class method: Reads in Repository.conf[:REPOSITORY_PERMISSION_FILE]
+    # Semi-private class method
     def self.__read_in_authz_file
       # Check if configuration is in order
-      if Repository.conf[:REPOSITORY_PERMISSION_FILE].nil?
-        raise ConfigurationError.new("Required config 'REPOSITORY_PERMISSION_FILE' not set")
+      if !MarkusConfigurator.markus_config_repository_admin?
+        raise NotAuthorityError.new('Unable to read authsz file:' \
+                                    'Not in authoritative mode!')
+      end
+      if MarkusConfigurator.markus_config_repository_permission_file.nil?
+        raise ConfigurationError.new("Required config " \
+                                     "'REPOSITORY_PERMISSION_FILE' not set")
       end
-      if !File.exist?(Repository.conf[:REPOSITORY_PERMISSION_FILE])
-        File.open(Repository.conf[:REPOSITORY_PERMISSION_FILE], "w").close() # create file if not existent
+      if !File.exist?(MarkusConfigurator.
+                        markus_config_repository_permission_file)
+        # create file if it doesn't exist
+        File.open(MarkusConfigurator.
+                    markus_config_repository_permission_file, "w").close
       end
       # Load up the Permissions:
       file_content = ""
-      File.open(Repository.conf[:REPOSITORY_PERMISSION_FILE], "r+") do |auth_file|
+      # Hound is complaining about the do block so I'm trying it with braces
+      File.open(MarkusConfigurator.markus_config_repository_permission_file, 
+                'r+') { |auth_file|
         auth_file.flock(File::LOCK_EX)
         file_content = auth_file.read()
         auth_file.flock(File::LOCK_UN) # release lock
-      end
+      } 
       return file_content
     end
 
-    # Semi-private class method: Writes out Repository.conf[:REPOSITORY_PERMISSION_FILE]
+    # Semi-private class method
     def self.__write_out_authz_file(authz_file_contents)
       # Check if configuration is in order
-      if Repository.conf[:IS_REPOSITORY_ADMIN].nil?
-        raise ConfigurationError.new("Required config 'IS_REPOSITORY_ADMIN' not set")
-      end
-      if Repository.conf[:REPOSITORY_PERMISSION_FILE].nil?
-        raise ConfigurationError.new("Required config 'REPOSITORY_PERMISSION_FILE' not set")
+      if !MarkusConfigurator.markus_config_repository_admin?
+        raise NotAuthorityError.new(
+          "Unable to write authsz file: Not in authoritative mode!")
       end
-      # If we're not in authoritative mode, bail out
-      if !Repository.conf[:IS_REPOSITORY_ADMIN] # Are we admin?
-        raise NotAuthorityError.new("Unable to write out repo permissions:  Not in authoritative mode!");
+
+      if MarkusConfigurator.markus_config_repository_permission_file.nil?
+        raise ConfigurationError.new("Required config " \
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.